Tests/Client/Unit Tests/WebSocket/WebSocketProviderMock.swift

```diff
@@ -2,125 +2,61 @@
 //  WebSocketProviderMock.swift
 //  StreamChatClientTests
 //
-//  Created by Alexey Bukhtin on 01/05/2020.
 //  Copyright © 2020 Stream.io Inc. All rights reserved.
 //
 
 import XCTest
 @testable import StreamChatClient
 
-final class WebSocketProviderMock: WebSocketProvider {
-    let request: URLRequest
+class WebSocketProviderMock: WebSocketProvider {
+    var request: URLRequest
+    
     var isConnected: Bool = false
-    let callbackQueue: DispatchQueue
-    weak var delegate: WebSocketProviderDelegate?
-    private var connectionId: String?
     
-    var failNextConnectCount = 0
-    let timeout = 3
-    private var timer: DispatchSourceTimer?
+    var callbackQueue: DispatchQueue
+    
+    var delegate: WebSocketProviderDelegate?
+    
+    convenience init() {
+        self.init(request: .init(url: URL(string: "test_url")!), callbackQueue: .main)
+    }
     
-    init(request: URLRequest, callbackQueue: DispatchQueue) {
+    required init(request: URLRequest, callbackQueue: DispatchQueue) {
         self.request = request
         self.callbackQueue = callbackQueue
     }
     
+    /// How many times was `connect()` called
+    var connectCalledCount = 0
```

Comment:
Can we group variables and functions? Comment makes it obvious anyway

Reply:
What do you mean?

Reply:
ping @b-onc

Reply:
Putting variables on top, functions after them, and not mixing the order